src/com/android/calendar/event/EditEventFragment.java

```diff
@@ -560,14 +560,15 @@ public void onPause() {
         Activity act = getActivity();
         if (mSaveOnDetach && act != null && !mIsReadOnly && !act.isChangingConfigurations()
                 && mView.prepareForSave()) {
-            mOnDone.setDoneCode(Utils.DONE_SAVE);
-            mOnDone.run();
+            //mOnDone.setDoneCode(Utils.DONE_SAVE);
```

Comment:
Do not comment out code, it can always be retrieved by looking at the previous git commit. Just remove these lines.